events_processor/config/kafka/consumer.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/twmb/franz-go/pkg/kgo"
 	"go.opentelemetry.io/otel/attribute"
 
-	tracer "github.com/getlago/lago/events-processors/config"
+	tracer "github.com/getlago/lago/events-processor/config"
```

Comment:
Should there be an underscore in `events-processor`?

Reply:
Good catch! Actually, to be more consistent with the "go approach" the folder should be name `events-processor` instead of `events_processor`

I will rename it in an other PR